ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/tree/member/repository/MemberRepository.java

```diff
@@ -3,14 +3,13 @@
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.tree.member.entities.Member;
 
 import java.util.List;
-import java.util.Optional;
 import java.util.stream.Stream;
 
 public interface MemberRepository {
 
 	Member saveLdesMember(Member member);
 
-	Optional<Member> getLdesMemberById(String id);
+	boolean memberExists(String id);
```

Comment:
minor comment: I prefer that functions start with a verb, because they represent an `action`

Reply:
Otherwise, like this improvement very much :-)